lib/buzzn/transactions/admin/meter/create_real.rb

```diff
@@ -24,7 +24,25 @@ def registers_schema(params:, **)
       elsif r[:id].nil?
         schema = Schemas::Transactions::Admin::Register::CreateMeta
       else
-        schema = Schemas::Transactions::Assign
+        begin
+          given_register = Register::Meta.find(r[:id])
+          name = r[:name]
+          label = r[:label]
+          unless name.nil? || label.nil?
```

Comment:
Consider adding a comment why a new meter has to be created if name  and label are empty and name and label do not match. Maybe the `unless` and the `if` block can be written in a cleaner way then.